lib/shoryuken/middleware/server/exponential_backoff_retry.rb

```diff
@@ -0,0 +1,43 @@
+module Shoryuken
+  module Middleware
+    module Server
+      class ExponentialBackoffRetry
+        include Util
+
+        def call(worker, queue, sqs_msg, body)
```

Comment:
Unused method argument - `queue`. If it's necessary, use `_` or `_queue` as an argument name to indicate that it won't be used.<br>Unused method argument - `body`. If it's necessary, use `_` or `_body` as an argument name to indicate that it won't be used.